apps/client/src/components/VisualStyles.js

```diff
@@ -7,6 +8,23 @@ export const FigCaption = styled.figcaption`
 export const Figure = styled.figure`
   min-height: 1px; /* IE11 Bug */
   margin: 0 0 24px 0;
+  ${({ isLoading, theme, animateLoading }) =>
+    isLoading &&
+    css`
+      ${perceivedLoading(animateLoading)}
+      height: 100px;
+      width: 100%;
+      & > * {
+        display: none;
+      }
```

Comment:
This piece also hides the FigCaption while the image is loading. I don't think this is what we want. Can you fix that?